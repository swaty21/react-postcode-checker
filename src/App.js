import React from 'react';
import PostCodeChecker from './screens/PostCodeChecker/index';

const App = () => {
  return (
    <>
      <PostCodeChecker />
    </>
  );
};

export default App;
