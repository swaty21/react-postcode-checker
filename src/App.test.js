/**
 * @jest-environment jsdom
 */
import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App Component', function () {
  it('should have hello world message', function () {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
