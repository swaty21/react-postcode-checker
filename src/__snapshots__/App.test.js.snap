// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should have hello world message 1`] = `
<div>
  <div>
    <h3>
       Please Enter Your Postcode
    </h3>
    <input
      data-testid="input-postcode"
      value=""
    />
    <button
      data-testid="submit"
      name="Submit"
      type="Submit"
    >
      Submit
       
    </button>
  </div>
</div>
`;
